tests/unit_tests/conftest.py

```diff
@@ -1054,7 +1061,7 @@ def config(self) -> Dict[str, Any]:
         if self._config is not None:
             return deepcopy(self._config)
 
-        self._config = {k: v["config"] for (k, v) in self._entries.items()}
+        self._config = {k: v.get("config") for (k, v) in self._entries.items()}
```

Comment:
@kptkin Double-check me on this? The problem was, [a couple tests](https://app.circleci.com/pipelines/github/wandb/wandb/15692/workflows/fcb77b9b-7be0-47a7-aad2-275f9a30bbb1/jobs/300547) were failing because... I think their runs must log artifacts, and the corresponding `resolve_{create,commit}_artifact` calls stuck stuff in `self._entries`, but those values didn't have `"config"` keys.

Reply:
Were they failing consistently? Sound more like some async stuff happening on the backend with artifact logging?.. Or some parsed data overwriting issues?..
The data parsed with resolvers gets merged into `self._entries[<run_id>]`. `resolve_upsert_bucket` should trigger when a run is created; that should always have config; now when the new artifact-related resolvers kick in, unless they overwrite config somehow, their stuff should just be recursively merged into the context. 
Could you please look into this more closely? We have a bunch of print statements commented out in conftest to help with debugging that exact thing, e.g. https://github.com/wandb/wandb/blob/cc526dfdf573b2040c14123fa2ef390dd0d82509/tests/unit_tests/conftest.py#L1502. Also, happy to pair code on this sometime soon!